cmd/build.go

```diff
@@ -19,11 +19,13 @@ var dotSlash = string([]byte{'.', filepath.Separator})
 var buildTargetMainDart string
 var buildTargetManifest string
 var buildTargetBranch string
+var buildCachePath string
 
 func init() {
 	buildCmd.Flags().StringVarP(&buildTargetMainDart, "target", "t", "lib/main_desktop.dart", "The main entry-point file of the application.")
 	buildCmd.Flags().StringVarP(&buildTargetManifest, "manifest", "m", "pubspec.yaml", "Flutter manifest file of the application.")
 	buildCmd.Flags().StringVarP(&buildTargetBranch, "branch", "b", "", "The go-flutter-desktop/go-flutter branch to use when building the embedder. (@master for example)")
+	buildCmd.Flags().StringVarP(&buildCachePath, "engineCache", "", "", "The path to save the flutter engine to (default is resolved to user cache)")
```

Comment:
Perhaps rename the flag to `engine-cache-path`, so that the name makes clear what kind of argument it expects.
(In any case, flag long name should use dash, not camelCase)

Reply:
Same for the flag in `hover run` cmd